common/sgx/quote.c

```diff
@@ -12,17 +14,18 @@
 #include <openenclave/internal/trace.h>
 #include <openenclave/internal/utils.h>
 #include "../common.h"
+#include "../tdx/quote.h"
 #include "collateral.h"
 #include "endorsements.h"
 #include "qeidentity.h"
 
 #include <time.h>
 
 #ifdef OE_BUILD_ENCLAVE
-#include "../enclave/sgx/verifier.h"
+#include "../../enclave/sgx/verifier.h"
 #include "platform_t.h"
 #else
-#include "../host/sgx/quote.h"
+#include "../../host/sgx/quote.h"
```

Comment:
how did thus work earlier :)

Reply:
One of the include paths we added made this work.